dev/integration_tests/ios_platform_view_tests/ios/PlatformViewUITests/PlatformViewUITests.m

```diff
@@ -26,16 +26,32 @@ - (void)setUp {
   [super setUp];
   self.continueAfterFailure = NO;
 
-  self.app = [[XCUIApplication alloc] init];
-  [self.app launch];
+  // Retry launching the app if it fails to launch.
+  // This is trying to fix a "failed to terminate" failure, which is likely a bug in Xcode.
+  // The solution is based on https://stackoverflow.com/questions/41872848/xctests-failing-to-launch-app-in-simulator-intermittently
+  int remainingLaunchCount = 10;
+  while (true) {
```

Comment:
for loop?

Reply:
I actually started with a for-loop, but found that I was checking the termination condition twice (once in the `for` and once in the inner `if`), which can easily make mistake. 

```
for (int i = 0; i < 10; i++) {
  // ...
  // ... 
  if (i < 9) { // this should be i < 9, not i < 10, i stared at it for like 30 seconds when writing it
    NSLog(@"Retry launch %d", i);
  } else {
    NSLog(@"Give up, failed to launch");
  }
```